rasa/core/policies/ensemble.py

```diff
@@ -578,16 +578,29 @@ def _pick_best_policy(
 
         form_confidence = None
         form_policy_name = None
-        # End-to-end predictions overrule all other predictions.
-        use_only_end_to_end = any(
+        # different type of predictions have different priorities
```

Comment:
And here we are again - having different priorities ~per~ within policies 😆

Reply:
yes